docs/csharp/language-reference/keywords/snippets/usings.cs

```diff
@@ -27,51 +27,59 @@ private static void UsingExample()
             using (var reader = new StringReader(manyLines))
             {
                 string? item;
-                do {
+                do
+                {
                     item = reader.ReadLine();
                     Console.WriteLine(item);
-                } while(item != null);
+                }
+                while (item != null);
```

Comment:
This is the only format I disagree with. I think putting the `while` on the next line makes it too easy to mis-read it as a stand-along `while` with a `;` as the statement. I'd consider the following instead:

```suggestion
                } while (item != null);
```

Reply:
I agree, and that's how it's shown in the [language reference article](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/statements/iteration-statements#the-do-statement)

Reply:
Sounds good, will update - thanks.